File: Aula 09/Exemplo04.java
import java.util.Scanner;
class Exemplo04 {
  public static void main(String[] args) {
    int soma = 0, acima = 0, n = 20;
    int idade[] = new int[n];
    Scanner sc = new Scanner(System.in);
    
    for(int i = 0; i < idade.length; i++){
      System.out.println("Digite a idade do aluno: ");
      idade[i] = sc.nextInt();
      soma = soma + idade[i];
    }
    sc.close();
    
    float media = soma / idade.length;
    for(int i = 0; i < idade.length; i++){
      if(idade[i] > media)
          acima++;
    }    
    System.out.println("A média da idade dos alunos é " + media);
    System.out.println("O número de alunos com idade acima da média é " + acima);
  }
}

File: Aula 09/Exemplo04.py
idade = []
acima = 0
soma = 0
n = 20

for i in range(n):
  num = int(input(f'Entre com a idade do {i+1}º aluno: ' )) 
  idade.append(num)
  soma = soma + num
  
media = soma / len(idade)

for i in range(n):
  if idade[i] > media:
      acima = acima + 1

print(f'A média de idade dos alunos é: {media:.2f}')
print(f'A quantidade de aluno com idade acima da média é de: {acima}')